components/rmrk/Create/AttributeTagInput.vue

```diff
@@ -1,59 +1,61 @@
 <template>
-  <NeoField :label="$t('Tags')">
-    <b-taginput
+  <NeoField :label="$t('Tags')" :addons="false">
+    <NeoInputitems
```

Comment:
```suggestion
    <NeoTagInput?
```

Reply:
Well Yes and no, more easy to find documentation about `InputItems` on oruga than `TagInput` which doesn't exist

Reply:
lmk if you are sure about this one